base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -193,25 +193,25 @@
     };
 
     static public final Integer[] clientECCiphers = {
-/*
-        SSLSocket.TLS_ECDH_RSA_WITH_AES_128_CBC_SHA,
-        SSLSocket.TLS_ECDH_RSA_WITH_AES_256_CBC_SHA,
-*/
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
-//        SSLSocket.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
-//        SSLSocket.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
-
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384,
-        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
-/*
-        SSLSocket.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
-        SSLSocket.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
-        SSLSocket.TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256
-*/
+            /*
+                    SSLSocket.TLS_ECDH_RSA_WITH_AES_128_CBC_SHA,
+                    SSLSocket.TLS_ECDH_RSA_WITH_AES_256_CBC_SHA,
+            */
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
+            //        SSLSocket.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+            //        SSLSocket.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
+
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+            /*
+                    SSLSocket.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
+                    SSLSocket.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+                    SSLSocket.TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+            */
```

Comment:
I think for something like this we want to use single-line comments:
```
static public final Integer[] clientECCiphers = {
        // SSLSocket.TLS_ECDH_RSA_WITH_AES_128_CBC_SHA,
        // SSLSocket.TLS_ECDH_RSA_WITH_AES_256_CBC_SHA,
        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
        ...
};
```

Reply:
Ack